test/Interop/Cxx/class/invalid-unsafe-projection-errors.swift

```diff
@@ -30,6 +30,7 @@ public func test(x: M) {
   x.test2()
   // CHECK: note: C++ method 'test3' that returns unsafe projection of type 'Ptr' not imported
```

Comment:
These error messages "C++ method 'test3' that returns unsafe projection of type 'Ptr' not imported" should also be changed. I think we can also combine that in to this PR. 

I think we can change this error message as "C++ method 'test2' is unavailable in Swift as it may return a pointer to memory that is destroyed"